src/tools/auth0/handlers/databases.js

```diff
@@ -49,7 +49,7 @@ export default class DatabaseHandler extends DefaultHandler {
         });
     }
 
-    return Reflect.get(this.client.connections, fn, this.client.connections);
+    return this.client.connections[fn].bind(this.client.connections)
```

Comment:
Binding the context of this function with `this.client.connections` is ultimately what fixes this issue. 

I make the extra step and swap `Reflect.get` with the array bracket syntax for clarity; especially after consulting with Adam, we deemed `Reflect` to be an obscure and confusing addition here.